scanners/boostsecurityio/modelscan/module.yaml

```diff
@@ -0,0 +1,22 @@
+api_version: 1.0
+
+id: boostsecurityio/modelscan
+name: modelscan
```

Comment:
This is the pretty name, `BoostSecurity Modelscan` or `Modelscan`
https://github.com/boostsecurityio/dev-registry/blob/e74faba46eb4e8b273c6987dbf39e22dc49d3ad4/scanners/boostsecurityio/brakeman/module.yaml#L4